Aztec/Classes/Public/TextKit/TextStorage.swift

```diff
@@ -14,29 +14,25 @@ public class TextStorage: NSTextStorage {
     typealias TextNode = Libxml2.TextNode
     typealias RootNode = Libxml2.RootNode
 
-    /// Element names for bold.
-    ///
-    /// - Note: The first element name is the preferred one.
-    ///
-    private static let elementNamesForBold = ["b", "strong"]
-
-    /// Element names for italic.
-    ///
-    /// - Note: The first element name is the preferred one.
-    ///
-    private static let elementNamesForItalic = ["em", "i"]
-
-    /// Element names for strikethrough.
-    ///
-    /// - Note: The first element name is the preferred one.
-    ///
-    private static let elementNamesForStrikethrough = ["s", "strike"]
-
-    /// Element names for underline.
-    ///
-    /// - Note: The first element name is the preferred one.
-    ///
-    private static let elementNamesForUnderline = ["u"]
+    enum ElementTypes: String {
+        case bold = "b"
+        case italic = "em"
+        case striketrough = "s"
+        case underline = "u"
+        case link = "a"
+
+        var equivalentNames: [String] {
```

Comment:
Please add a description of why this exists, because even though it's fairly obvious to us, a newcomer could find this confusing.